Components/MenuCard.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {View, Text, StyleSheet, Touchable, Pressable} from 'react-native';
+
+const Main = props => {
+  // console.log('Menu Card props-', props);
+  const itemName = props.x.name;
```

Comment:
For deeply nested objects, we can use null checks like
```suggestion
  const itemName = props?.x?.name;
```